blog-recommendation-ts/README.md

```diff
@@ -2,10 +2,11 @@
 
 This is a example application showcasing a recommendation system for blogs. Please make sure you've followed prerequiste instructions from the [parent README](../README.md) before following steps below
 
-⚠️ **Note**: This requires a canary build of the `js2wasm` plugin. Use the following command to install it:
+⚠️ **Note**: This requires v0.6.0 or later of the `js2wasm` plugin. Use the following commands to install it:
```

Comment:
They might have to delete canary if they are on it as it seems to be considered a higher version number.

Reply:
Do you think we should add a note about this here in the READMEs as well?

Reply:
Not quite sure if necessary but I think it would be better no?

Reply:
Ok, updated